frontend/src/pages/IstioConfigDetails/IstioObjectDetails/IstioConfigOverview.tsx

```diff
@@ -94,6 +105,11 @@ class IstioConfigOverview extends React.Component<IstioConfigOverviewProps> {
       </div>
     );
 
+    let urlInKiali = "";
+    if (istioObject !== undefined && istioObject.metadata.namespace !== undefined && istioObject.kind !== undefined) {
+      urlInKiali = GetIstioObjectUrl(istioObject.metadata.name, istioObject.metadata.namespace , istioObject.kind.toLowerCase() );
```

Comment:
In my env this is failing, as it doesn't take into account the URL prefix; i.e. it is building something like `https://domain/namespaces/bookinfo/istio/virtualservices/productpage` which at first view looks OK, but clicking on it generates a `404 page not found` message, because the right link should be `https://domain/console/namespaces/bookinfo/istio/virtualservices/productpage` (notice the `console` after the domain)

I didn't notice this on my previous review :-/

Reply:
@israel-hdez is this happening on plugin console side only or also in Kiali itself?
I follow this section while running console locally https://github.com/kiali/openshift-servicemesh-plugin#how-to-run-the-plugin-for-local-development

Reply:
Now I see @israel-hdez , fixed it.